Sources/Build/LLBuildCommands.swift

```diff
@@ -76,7 +76,6 @@ final class TestDiscoveryCommand: CustomLLBuildCommand, TestBuildCommand {
 
                 fileprivate extension \#(className) {
                     @available(*, deprecated, message: "Not actually deprecated. Marked as deprecated to allow inclusion of deprecated tests (which test deprecated functionality) without warnings")
-                    @MainActor
                     static let __allTests__\#(className) = [
```

Comment:
If the types here are non-sendable, we may need to tag this variable as `nonisolated(unsafe)`.

Reply:
I would have expected so, but it doesn't seem to have made any difference.